ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -63,30 +84,65 @@ open class Tooltip: NSObject {
     private var gestureView: UIView?
     private var dismissMode: DismissMode = .tapAnywhere
 
-    private override init() { }
+    private override init() {
```

Comment:
`    @objc public static let defaultScreenMargins = UIEdgeInsets(top: Constants.defaultMargin, left: Constants.defaultMargin, bottom: Constants.defaultMargin, right: Constants.defaultMargin)`

this is interesting. is this being deprecated for tokens?

Reply:
not sure what you mean, can you elaborate?

Reply:
before in fluent1, we had exposed bunch of property to customize tooltip. if we are moving to token system, and we think this is something we continue to support as customization, we should move over?

Reply:
FYI: This is API breaking because we'll need to remove screenMargins parameter from show methods. Is that okay, @harrieshin?

Reply:
i think we should have deprecated declare so we can slowly migrate people over. do you happen to know who uses this btw?

Reply:
sounds good and not extensively, I only know of its usage in CocoaUI for teaching callouts